controllers/controller.go

```diff
@@ -17,6 +17,7 @@ const (
 	CPUSaturationLevel            = "konsumerator.lwolf.org/cpu-saturation-level"
 	ScalingStatusAnnotation       = "konsumerator.lwolf.org/scaling-status"
 	ScalingStatusChangeAnnotation = "konsumerator.lwolf.org/scaling-status-change"
+	KonsumeratorManagerAnnotation = "konsumerator.lwolf.org/managed"
```

Comment:
`ManagedByAnnotation = "konsumerator.lwolf.org/managed-by"`

Reply:
Changed to `ManagedByLabel`.